app/common/components/resource/resource-list/resource-list-config.factory.js

```diff
@@ -46,9 +46,10 @@ function resourceListConfig ($filter, table) {
     constructor (resource) {
       const textPhysical = utils.Naming.humanize(resource.physical || '')
       const textTrading = utils.Naming.humanize(resource.trading || '')
+      const textUsage = utils.Naming.humanize(resource.usage || '')
       const textAllocated = resource.allocated ? utils.Naming.humanize('allocated') : null
```

Comment:
@cayop: maybe the following line 50 can be removed, since allocate is already part of usage, right?
` const textAllocated = resource.allocated ? utils.Naming.humanize('allocated') : null`